python/ctsm/config_utils.py

```diff
@@ -50,7 +50,7 @@ def get_config_value(
     is_list=False,
     convert_to_type=None,
     can_be_unset=False,
-):
+):  # pylint: disable=too-many-positional-arguments
```

Comment:
I wonder about upping the limit from 5 to something a little higher in the .pylintrc file? Allowing too many encourages making complex arguments and subroutines that are too big. So it's good to have.

But, maybe 9 or 10 for the limit instead of 5? When we override it with a pylint: disable we should also comment to say if it's something to fix, or it's OK in this case.

Reply:
All either resolved or annotated by 854ddd8ad.